### DISCUSSION TITLE
Is it possible to implement this form builder into my existing Laravel project

### DISCUSSION THREAD

**Discussion by aazimabdllh** [OpnForm/OpnForm#133] [Q&A]
I want to add a form builder for my existing Laravel project, and this OpnForm is exactly what I needed. I tried to clone this repo and run on my local, it works perfectly. But my only concern is, I already have a working Laravel application and I want to add this form builder into the admin page, so that admin can configure and embed the form to my other Laravel application. I'm not so familiar with Vue.js framework, that's why I'm thinking to just get the form builder part, instead of the whole project folder.

**Comment by JhumanJ** [comment]
HI, this is a standalone project, not a package.
